### PR TITLE
P1 features

### PR DESCRIPTION
**Pull request by Rohitprasad83**
# added Todos
- add a todo
- delete a todo
- mark a todo as completed

chrome_8C5354nQ8M

### CODE REVIEW

**Review by Sharmaryan — APPROVED** [review]
-UI is looking nice
- code is clean and easy to understand

File: src/components/Todo.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <div className=" flex flex-col items-start gap-2 bg-slate-800 w-max absolute bottom-12 right-4 p-2 rounded-md">
+          <p className="cursor-pointer">Today</p> <span></span>
+          <div className="overflow-y-auto max-h-96">
+            {todos.map(todo => (
+              <div
+                key={todo.id}
+                className="flex items-center w-40"
+                style={
+                  todo.completed
+                    ? { textDecoration: 'line-through' }
```
Comment (parthpandyappp) [inline src/components/Todo.jsx:57]: Instead of adding a style attribute, toggle the classNames using template literals.

**Comment by parthpandyappp** [comment]
- The code looks clean.
- The UI is well designed.

Added some comments, have a look at it :)

**Review by parthpandyappp — APPROVED** [review]

File: src/components/Todo.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    setTodos(updatedTodos)
+  }
+
+  useEffect(() => {
+    const todos = JSON.parse(localStorage.getItem('todos'))
+    todos ? setTodos(todos) : setTodos([])
+  }, [])
+
+  return (
+    <div className="">
```
Comment (irajbhinde) [inline src/components/Todo.jsx:46]: You can remove the empty classname

File: src/components/Todo.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      {openTodo && (
+        <div className=" flex flex-col items-start gap-2 bg-slate-800 w-max absolute bottom-12 right-4 p-2 rounded-md">
+          <p className="cursor-pointer">All Todos</p> <span></span>
```
Comment (irajbhinde) [inline src/components/Todo.jsx:49]: There's an empty span tag present here, does it have any significance?

File: src/components/Todo.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                  <i
+                    className="fa-solid fa-trash cursor-pointer"
+                    onClick={() => deleteTodo(todo)}></i>
+                </div>
+              </div>
+            ))}
+          </div>
+          <input
+            type="text"
+            name=""
```
Comment (irajbhinde) [inline src/components/Todo.jsx:79]: You can remove the empty name attribute

**Comment by irajbhinde** [comment]
The code looks neat, just remove the unused attributes! Overall, looks good to merge

**Pull request merged**